src/app/pages/Languages.svelte

```diff
@@ -124,6 +133,11 @@
 <main>
   <div class="languages">
     <div class="languages__container">
+      <Link to="/">
```

Comment:
I think these link elements could be why the test runner is complaining, though I'm not sure why it's only complaining on the runner. 

Three potential fixes in my head could be:
- Do as the runner says and try `npm install @types/svelte-routing` if it exists or add a new declaration (.d.ts) file containing `declare module 'svelte-routing';` (ts)
- Use `<a>` tags with `href`s. Svelte seems to play nicely with these
- Something else entirely

Let me know what the fix is! I'm curious about why the runner's failing

Reply:
`yarn add @types/svelte-routing` please!

Reply:
I'm really confused because we're using `svelte-routing` elsewhere, and TypeScript does not complain about it.

A workaround is using `<a href="/">` since this _particular_ `<Link>` tag is semantically identical to that.